### PR TITLE
DM-38449: Updated templates to goodSeeing, removed old deep

### PR DESCRIPTION
**Pull request by erinleighh**

### CODE REVIEW

File: scripts/generate_templates_gen3.sh [hunk 1]
```diff
@@ -41,16 +41,17 @@ set -e
```
Comment (kfindeisen) [inline scripts/generate_templates_gen3.sh:41]: Do you have an update for the runtime estimate on line 24? (I'd imagine not, if you never ran this script in its current form, but I wanted to check.)

File: scripts/import_templates_gen3.py [hunk 2]
```diff
@@ -87,8 +87,9 @@ def _export(butler, export_file):
     runs : iterable [`str`]
         The names of the runs containing exported templates.
     """
-    skymaps = butler.registry.queryDataIds("skymap", datasets=TEMPLATE_NAME, collections=butler.collections)
-    skymap_query = f"skymap IN ({', '.join(repr(id['skymap']) for id in skymaps)})"
+    skymaps = butler.registry.queryDataIds("skymap", datasets=TEMPLATE_NAME, collections=TEMPLATE_COLLECT)
```
Comment (kfindeisen) [inline scripts/import_templates_gen3.py:90]: Why the change from `butler.collections` to `TEMPLATE_COLLECT`? I wouldn't expect `templates/goodSeeing` to exist in the source repository, in general.

File: scripts/import_templates_gen3.py [hunk 3]
```diff
@@ -87,8 +87,9 @@ def _export(butler, export_file):
...
+    #skymap_query = f"skymap IN ({', '.join(repr(id['skymap']) for id in skymaps)})"
+    skymap_query = " or ".join(f"skymap = '{id['skymap']}'" for id in skymaps)
```
Comment (kfindeisen) [inline scripts/import_templates_gen3.py:92]: Please remove the vestigial line.

Comment (kfindeisen) [reply scripts/import_templates_gen3.py:92]: Since `IN` will never work with governor dimensions like `skymap` (IIRC that was why this change was needed?), can you also update [the script in `ap_verify_dataset_template`](https://github.com/lsst-dm/ap_verify_dataset_template/blob/main/scripts/import_templates.py#L100) with the working form?

**Review by kfindeisen — APPROVED** [review]
The changes to the dataset look good. My main concern is the use of `templates/goodSeeing` as an input rather than an output collection in the import script; any clarification would be appreciated.

File: scripts/import_templates_gen3.py (hunk 3 above)
Comment (timj) [reply scripts/import_templates_gen3.py:92]: I've discussed this problem with @TallJimbo and we were not expecting it to trigger without any other constraints. We will file a ticket to fix the problem with `IN` such that `bind` with a list could also be used without having to form the string query yourself with `join`.

File: scripts/import_templates_gen3.py [hunk 4]
```diff
@@ -52,8 +52,8 @@
 
 def _make_parser():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-b", dest="src_dir", default="/repo/main",
-                        help="Repo to import from, defaults to '/repo/main'.")
+    parser.add_argument("-b", dest="src_dir", default="/sdf/group/rubin/repo/main",
+                        help="Repo to import from, defaults to '/sdf/group/rubin/repo/main'.")
```
Comment (kfindeisen) [inline scripts/import_templates_gen3.py:56]: Please include this change in [`ap_verify_dataset_template`](https://github.com/lsst-dm/ap_verify_dataset_template/blob/main/scripts/import_templates.py#L62-L63) as well.

File: scripts/import_templates_gen3.py (hunk 2 above)
Comment (erinleighh) [reply scripts/import_templates_gen3.py:90]: Thank you for catching that! This was a mistake while trying to get the HSC version of this script to work. I was on the wrong tab, and I guess I didn't undo enough times.

File: scripts/generate_templates_gen3.sh (hunk 1 above)
Comment (erinleighh) [reply scripts/generate_templates_gen3.sh:41]: No runtime estimate, but it might be good to test this script sometime in the future. I'll make a note of it!
